core/src/main/java/org/elasticsearch/common/inject/internal/ConstructionContext.java

```diff
@@ -109,12 +109,10 @@ public Object invoke(Object proxy, Method method, Object[] args)
             try {
                 // This appears to be not test-covered
                 return method.invoke(delegate, args);
-            } catch (IllegalAccessException e) {
-                throw new RuntimeException(e);
-            } catch (IllegalArgumentException e) {
-                throw new RuntimeException(e);
-            } catch (InvocationTargetException e) {
+            } catch(InvocationTargetException e) {
                 throw e.getTargetException();
+            } catch(IllegalAccessException | IllegalArgumentException e) {
```

Comment:
Can you add a space between `catch` and `(` throughout?

Reply:
of course.